src/Microsoft.Azure.EventHubs.Processor/Options/JsonConvertOptions.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections.Generic;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace Microsoft.Azure.EventHubs.Processor.Options
+{
+    /// <summary>
+    /// JsonConvert Options
+    /// </summary>
+    public static class JsonConvertOptions
+    {
+        /// <summary>
+        /// Get the JsonConvert formatting options. so it's not conflicting / dependent upon the user's code.
+        /// </summary>
+        /// <returns></returns>
+        public static JsonSerializerSettings GetOptions()
```

Comment:
>JsonSerializerSettings [](start = 22, length = 22)

Allow custom settings as well. Like getter/setter, allow to customize their casing. If not set, return the default.